### DISCUSSION TITLE
RLC-1212A

### DISCUSSION THREAD

**Discussion by WMP** [AT0myks/reolink-fw-archive#22] [Firmware sharing]
I bought new camera and i have firmware:
Model:RLC-1212A
Build No.: build 22090715
Hardware No.: IPC_523B18128M12MP
Config Version: v3.1.0.0
Firmware Version: v3.1.0.1294_22090715
Details: IPC_523B18128M12MPS18E1W0110000

**Comment by AT0myks** [comment]
Thanks for the info, I added it to the list of missing firmwares in 36fe2f932de97551b685fb150228f6654a74e5fd. A more recent firmware v3.1.0.2174_23050815 is available in case you haven't seen yet.
